Sources/EmbraceCore/Capture/Network/URLSessionCaptureService+Options.swift

```diff
@@ -1,12 +1,13 @@
 //
-//  Copyright © 2023 Embrace Mobile, Inc. All rights reserved.
+//  Copyright © 2024 Embrace Mobile, Inc. All rights reserved.
 //
 
 import Foundation
 
 extension URLSessionCaptureService {
     /// Class used to setup a URLSessionCaptureService.
-    @objc public final class Options: NSObject {
+    @objc(URLSessionCaptureServiceOptions)
```

Comment:
Missing the `EMB` prefix
```suggestion
    @objc(EMBURLSessionCaptureServiceOptions)
```